deployments/ecmps/release-v2.0-fix/Sprint8/6111/camdecmps/functions/rpt_quartely_em_summary_data.sql

```diff
@@ -0,0 +1,331 @@
+                    and  exists
+                            (
+                                select  1
+                                from   (
+                                            select  dat.mon_loc_id,
+                                                    (
+                                                        select  case
+                                                                    when sub.calendar_year = lst.calendar_year then
+                                                                        case
+                                                                            when sub.quarter < 2
+                                                                            then 2
+                                                                            else sub.quarter
+                                                                        end
+                                                                    else 2
+                                                                end
+                                                        from  camdecmpsmd.reporting_period sub
+                                                        where  sub.Begin_Date = dat.First_Begin_Date
+                                                    ) as First_Quarter
+                                            from  (
+                                                        select  mpl.mon_loc_id,
+                                                                min( prd.Begin_Date ) as First_Begin_Date
+                                                        from  camdecmps.monitor_plan_location mpl
+                                                                join camdecmps.monitor_plan pln
+                                                                on pln.mon_plan_id = mpl.mon_plan_id
+                                                                join camdecmpsmd.reporting_period prd
+                                                                on prd.rpt_period_id = pln.Begin_Rpt_Period_Id
+                                                        where  mpl.mon_loc_id = lst.mon_loc_id
+                                                        group
+                                                            by  mpl.mon_loc_id
+                                                    ) dat
+                                        ) agg
+                                where  ( agg.First_Quarter <= prd.quarter )
+                            )
+                    group
+                        by  lst.mon_plan_id,
+                            lst.mon_loc_id,
+                            lst.calendar_year,
+                            osd.fuel_cd
+                    union   all
+                    --
+                    -- Year-to-Date Data for the Non-Fuel-Specific Report Tables
+                    --
+                    select  lst.mon_plan_id,
+                            0 as fuel_ind,
```

Comment:
Since FUEL_IND is not needed in this query because it is always 0, and because it is not needed in the sub-queries with which it is unioned.